### ISSUE TITLE
Black screen when running on FreeDOS 1.3 on WMware player

### ISSUE THREAD

**Issue by berik99** [SuperIlu/DOjS#29]
I'm trying to run the example 'exampl.js', but when i run it the screen goes black and the machine stops responding and the only solution is to restart.

**Comment by SuperIlu** [comment]
Several questions:
- how did you try to run exampl.js? can you provide the command line?
- does the black screen happen with the editor or the sketch view?
- after restart: is there a jslog.txt file? Can you provide it here?
- I never used VMware player, can you provide a screenshot of the HW settings of the WM?

**Comment by berik99** [comment]
- The command that i used is `dojs.exe example\exampl.js` from DOjS folder
- The black screen appears either by running the script directly via the -r flag, or by running it from the editor
- I downloaded the FreeDOS iso from the [official website](https://freedos.org/download/) of the project and installed DOjS from the official FreeDOS repository (FDNPKG)
- The jslog.txt: [JSLOG.TXT](https://github.com/SuperIlu/DOjS/files/9653234/JSLOG.TXT)
- WMware Settings: [wmware settings.zip](https://github.com/SuperIlu/DOjS/files/9653245/wmware.settings.zip)

I tried on VirtualBox and with the same settings it works fine.

**Comment by SuperIlu** [comment]
Thanks, I'll try to reproduce when I find some time.
Up to now I never checked DOjS on VMware, only VBox, DOSBox and a bunch of real HW...

**Comment by SuperIlu** [comment]
I did a quick dive and installed VMware and FDDOS 1.3 in default configuration for MS-DOS.

I was able to reproduce with **VMware Workstation 16 Player** and **DOjS 1.7.0** from the FDOS 1.3 Bonus CD.

- Removing the network adapter and the sound card did change nothing
- reducing the memory to 64MiB did change nothing
- `dojs.exe -r -w 320 -b 8 example\exampl.js` works (320x200 with 8 bits color depth)
- all other resolution are either "not available" or crash the VM

For now I have to assume an incompatibility between the Allegro display driver and the VGA/VESA emulation of VMware.
I will take a deeper look into this another time, but if I can confirm that this will sadly be a "won't fix" issue.
I lack knowledge and resources to dive into Allegros display driver...
